code/modules/tables/interactions.dm

```diff
@@ -85,6 +85,8 @@
 			if(occupied)
 				user << "<span class='danger'>There's \a [occupied] in the way.</span>"
 				return
+			if(!user.Adjacent(M))
+				return

```

Comment:
This part is a bugfix, not a feature change. Just going to point that out to everyone in protest.